scenarios/mnist/deployment/docker/docker-compose-train.yml

```diff
@@ -0,0 +1,12 @@
+services:
+  train:
+    image: ${CONTAINER_REGISTRY}/depa-training:latest
+    volumes:
+      - $ICMR_INPUT_PATH:/mnt/remote/icmr
```

Comment:
These are not correct. Please make sure we have no references to icmr, cowin and index under the mnist folder.